### DISCUSSION TITLE
How to compile multiple, unrelated CSS files, in a single webpack run?

### DISCUSSION THREAD

**Discussion by binarykitchen** [webpack/webpack#18617] [General]
In short:
I have a full-stack project, including email templates and their own styles. How to include them in webpack, when the entry is a single TSX file?

Longer version, in gulp, you can do something like this:
```
  gulp
    .src(["main.styl", "email.styl"]) // main.styl is used for the frontend part, email.styl not but needed for backend to style emails
    .pipe(
      gulpStylus(),
    ).
    .pipe(gulpRev())    
    ...
```

Like that can be injected into the app after build. Wondering if webpack can do that as well?
There are plenty of webpack hooks. Possible to compile a single, unrelated CSS separately to the main app like `email.styl` in the above example?

**Comment by alexander-akait** [comment]
You can use them as entrypoints, sorry I find it difficult to understand what exactly you mean

**Reply by binarykitchen** [reply]
Yes, almost.

This thread helped: https://github.com/webdiscus/html-bundler-webpack-plugin/discussions/104
